packages/block-editor/src/components/block-list/block-mobile-floating-toolbar.native.scss

```diff
@@ -22,3 +24,7 @@ $floating-toolbar-height: 44;
 .floatingToolbarFillColorDark {
 	background-color: rgba(#3c434a, 0.85);
 }
+
+.hitslop {
+	top: $floating-toolbar-shift;
```

Comment:
I added this as a way to share the value of the constants to the JS code because I couldn't find a better pattern. Is there a better way to share constants between CSS and JS?